emr_launcher.py

```diff
@@ -101,12 +131,14 @@ def launch_cluster(event: dict = {}, context: object = None) -> dict:
     cluster_config.update(read_config("steps", False))
     logger.debug("Requested cluster parameters", extra=cluster_config)
 
-    # logger.info("Submitting cluster creation request")
-    # emr = boto3.client("emr")
-    # resp = emr.run_job_flow(**cluster_config)
-    # logger.info("Cluster submission successful", extra=resp)
+    logger.info("Submitting cluster creation request")
+    emr = boto3.client("emr")
+    resp = emr.run_job_flow(**cluster_config)
+    logger.info("Cluster submission successful", extra=resp)
+
+    print(resp)
```

Comment:
Can we change this to a `logger.debug()` call please?

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/23cce78917a26136ad736a21750d7520d4e711d1